### PR TITLE
Add Info About Tokens in README

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
I added a small section about the token retrieval and permissions it has to have. Also I formatted a README a bit.

Closes: #26.

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you take a look, please?

File: README.md [hunk 1]
```diff
@@ -23,17 +23,57 @@ $ mergem --repo yegor256/mergem --verbose --token <YOUR_GITHUB_TOKEN>
...
+those pull requests that are coming not
+from [Renovate](https://github.com/apps/renovate)
+or [Dependabot](https://github.com/dependabot). Then, it will
+post `@rultor merge` text message to each pull request left in the list.
+
+## Token
+
+`mergem` requires a GitHub token to be passed via `--token` option. In order to
+receive one, go to your GitHub account, then to "Settings", then to "Developer
+Settings" (or just use the ).
```
Comment (yegor256) [inline README.md:36]: @volodya-lombrozo typo?

Comment (volodya-lombrozo) [reply README.md]: @yegor256 Yes. Thank you!

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you have a look one more time, please?

**Comment by yegor256** [comment]
@volodya-lombrozo thanks!

**Pull request merged**